### ISSUE TITLE
Upgraded phone, old client ID still being used. Push no longer working

### ISSUE THREAD

**Issue by mannibis** [cocodelabs/znc-palaver#19]
Hi, I just upgraded from an iPhone 5 to iPhone 6, unloaded and reloaded the ZNC module, but the client ID still remains the same (the old one). test command does not work, and list only shows the old client ID from my iPhone 5. How do I update the client ID for my new iPhone 6. The palaver app is installed and connected to my ZNC server already. Things are working on the app end, just not the ZNC push module. Thanks!

**Comment by kylef** [comment]
Hi @mannibis, 

I have a feeling this is a bug on iOS where the operating system doesn’t issue a new push notification token to applications when the old one becomes invalid (due to a restore).

Could you try going to the system settings on your iOS device, and then scrolling into “Palaver”. Then going into “Notifications” then toggle “Allow Notifications” off, and back on. Once you launch Palaver again, iOS should issue a new push notification token to Palaver (and then subsequently your ZNC server).

Please let me know if this fixes the problem.

**Comment by mannibis** [comment]
Hey kylef. Thanks for your response...after a bunch of stuff that I tried (including removing ~/.znc/moddata/palaver.conf), I eventually did exactly what you said (Notifications > Palaver > ON and OFF) and did /msg *palaver list,and lo and behold both tokens showed up. Can't seem to get rid of the first, but push is working again (tho it sends to both). I did exactly what you said, but on a whim. However, now I'm afraid if I unload/reload the module, it won't work because I deleted the conf file, hoping it would be rebuilt. Either way, it works! Thanks!
